src/options/components/navigation/index.jsx

```diff
@@ -1,62 +1,55 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Link } from 'react-router'
-import classNames from 'classnames'
-
-import styles from './styles.css'
-
-const Navigation = ({ currentLocation, routes }) => {
-    function buildRoutes() {
-        return routes.map((route, idx) => {
-            const navClasses = classNames({
-                [styles.navLink]: true,
-                [styles.isActive]: isActive(route),
-            })
-
-            const navIcon = classNames({
-                [styles.navIcon]: true,
-                'material-icons': true,
-            })
-
-            return (
-                <li className={navClasses} key={idx}>
-                    <i className={navIcon}>{route.icon}</i>
-                    {route.component === 'faq' && (
-                        <a
-                            className={navClasses}
-                            href={route.pathname}
-                            target="_blank"
-                        >
-                            {route.name}
-                        </a>
-                    )}
-                    {route.component !== 'faq' && (
-                        <Link className={navClasses} to={route.pathname}>
-                            {route.name}
-                        </Link>
-                    )}
-                </li>
-            )
-        })
+import { connect } from 'react-redux'
+import * as selectors from '../../imports/selectors'
+import * as actions from '../../imports/actions'
+import { bindActionCreators } from 'redux'
+
+import Nav from './Nav'
+
+class Navigation extends Component {
+    render() {
+        const { routes } = this.props
+        const { isRunning, isIdle, isLoading, isStopped, isPaused } = this.props
+        const state = {
+            isRunning: isRunning,
+            isIdle: isIdle,
+            isLoading: isLoading,
+            isStopped: isStopped,
+            isPaused: isPaused,
+        }
+
+        return (
+            <Nav
```

Comment:
If you update `Nav` view to have just `children` props, you would pass in `children` props like this (assuming there is a class method called `renderNavLinks`):

```jsx
return (
  <Nav>
    {this.renderNavLinks()}
  </Nav>
)
```